Seqs_processing_tool/bio_files_processor.py

```diff
@@ -0,0 +1,37 @@
+def convert_multiline_fasta_to_oneline(input_fasta: str, output_fasta: str) -> None:
+    """
+    Description of convert_multiline_fasta_to_oneline
+
+    :param input_fasta: path to fasta file.
+    :param output_fast: name of the new file.
+    :return: None
+    """
+    with open(input_fasta) as fasta_file:
+        with open(output_fasta, "w") as multiline_fasta_file:
+            multiline_fasta_file.write(fasta_file.readline())
+            for line in fasta_file:
+                if line.startswith(">"):
```

Comment:
Again, very streamlined code, thank you. The only thing bothering me - rewriting original file might not be the best solution.